Assets/Scripts/API/BiogFile.cs

```diff
@@ -0,0 +1,434 @@
+                }
+                // Social group (Merchants, Commoners, etc.)
+                else
+                {
+                    if (!int.TryParse(tokens[0].Split('r')[1], out id) || !int.TryParse(tokens[1], out amount))
+                    {
+                        Debug.LogError("CreateCharBiography: r - invalid argument.");
+                        return;
+                    }
+                    playerEntity.SGroupReputations[id] += (short)amount;
+                }
+            }
+            // Adjust poison resistance
+            else if (effect.StartsWith("RP"))
+            {
+                if (!int.TryParse(tokens[1], out parseResult))
+                {
+                    playerEntity.BiographyResistPoisonMod = parseResult;
+                }
+                else
+                {
+                    Debug.LogError("CreateCharBiography: RP - invalid argument.");
+                }
+            }
+            // Adjust fatigue
+            else if (effect.StartsWith("FT"))
+            {
+                if (!int.TryParse(tokens[1], out parseResult))
+                {
+                    playerEntity.BiographyFatigueMod = parseResult;
+                }
+                else
+                {
+                    Debug.LogError("CreateCharBiography: FT - invalid argument.");
+                }
+            }
+            // Adjust reaction roll
+            else if (effect.StartsWith("RR"))
+            {
+                if (!int.TryParse(tokens[1], out parseResult))
+                {
+                    playerEntity.BiographyReactionMod = parseResult;
+                }
+                else
+                {
+                    Debug.LogError("CreateCharBiography: RR - invalid argument.");
+                }
+            }
+            // Adjust disease resistance
+            else if (effect.StartsWith("RD"))
+            {
+                if (!int.TryParse(tokens[1], out parseResult))
+                {
+                    playerEntity.BiographyResistDiseaseMod = parseResult;
+                }
+                else
+                {
+                    Debug.LogError("CreateCharBiography: RD - invalid argument.");
+                }
+            }
+            // Adjust magic resistance
+            else if (effect.StartsWith("MR"))
+            {
+                if (!int.TryParse(tokens[1], out parseResult))
+                {
+                    playerEntity.BiographyResistMagicMod = parseResult;
+                }
+                else
+                {
+                    Debug.LogError("CreateCharBiography: MR - invalid argument.");
+                }
+            }
+            // Adjust to-hit
+            else if (effect.StartsWith("TH"))
+            {
+                // TODO: Apply to-hit modifier when implemented
+                Debug.Log("CreateCharBiography: Biography to-hit modifier not yet implemented.");
+            }
```

Comment:
This is the `playerEntity.BiographyAvoidHitMod`. I named it that because that's what it actually does, help the player avoid hits by enemies.